src/components/EpisodePreview/styles.scss

```diff
@@ -1,5 +1,39 @@
-.wrapper {
-	&[aria-hidden="true"]{
-		opacity: 0;
-	}
+.container {
+	width: 100%;
 }
+
+	.thumbnail {
+		display: flex;
+		position: relative;
+		width: 100%;
+		height: 38vh;
+		background-color: blue;
+
+		a { //need classname
+			display: flex;
+			width: 100%;
+			height: 100%;  //width/height 100% for everything!???
+		}
+	
+			img { //need classname
+				width: 100%;
+				height: 100%;
+			}
+
+			svg { //need classname
+				width: 17.5%;
+				height: 25%;;
```

Comment:
I think SVGs have scaling built into them, so you can specify the general size you'd like them to be contained within and they'll scale themselves up as much as possible whilst maintaining their aspect ratio...although you might need to put `preserveAspectRatio="true"` on them, or something? Worth looking-into and having a fiddle.

Reply:
Made some changes, not perfect but better!